recipes/kwant/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "kwant" %}
+
+package:
+  name: {{ name }}
+  version: "1.2.2" [py3k]
+  version: "1.1.2" [py27]
+
+source:
+  git_url: https://gitlab.kwant-project.org/kwant/kwant.git
+  git_tag: v1.2.2 [py3k]
+  git_tag: v1.1.2 [py27]
```

Comment:
@basnijholt change the top of your recipe to

```yaml
{% set name = "kwant" %}
{% set py3k_ver = "1.2.2" %}
{% set py2k_ver = "1.1.2" %}

package:
  name: {{ name }}
  version: {{ py3k_ver }}  # [py3k]
  version: {{ py2k_ver }}  # [py2k]

source:
  fn: v{{ version }}.tar.gz
  url: https://gitlab.kwant-project.org/kwant/kwant/archive/v{{ version }}.tar.gz
  sha256: 8c20230f266622e5fa4c75fd3b1e8018d9a99643e71b86a7e91cb7b6902d8620  # [py3k]
  sha256: 3229009580e33e58f5cfb8405c44f8146b89b621b6ebe599166ef9add8ec9755  # [py2k]
```

That also fixed the linter issue locally for me.  BTW do the different version reflect the Python versions? If so maybe adding a `py3k` version first and than a maintenance branch might be a better strategy.